Sources/Fuzzilli/FuzzIL/Program.swift

```diff
@@ -32,18 +32,24 @@ public final class Program: Collection {
 
     /// Result of runtime type collection execution, by default there was none
     public var typeCollectionStatus = TypeCollectionStatus.notAttempted
-    
+
+    public var lastVariable: Int {
+        return instructions.reduce(0, {cur, instr in
+            cur + instr.allOutputs.count
```

Comment:
Maybe this function should be called `nextFreeVariable` since this function currently returns 0 for an empty program, which would be incorrect?
Also maybe you can compute the value without traversing all instructions with something like
```
for instr in instructions.reversed() {
    if let r = instr.allOutputs.max() {
        return r.number + 1
    }
}
return 0
```
WDYT?

Reply:
Ah you're right, we should call it `nextFreeVariable` :)